packages/flutter_tools/lib/src/version.dart

```diff
@@ -702,6 +702,7 @@ class GitTagVersion {
   final String hash;
 
   static GitTagVersion determine(ProcessUtils processUtils, [String workingDirectory]) {
+    _runGit('git fetch https://github.com/flutter/flutter --tags', processUtils, workingDirectory);
```

Comment:
Would you mind making the repo URL a constant and adopting in `fetchRemoteFrameworkCommitDate()`? https://github.com/flutter/flutter/blob/323e8c17edc086b5a36b5c1afed8154a364d9e2e/packages/flutter_tools/lib/src/version.dart#L236

Reply:
Done